cookbooks/aws-parallelcluster-slurm/recipes/config_head_node.rb

```diff
@@ -82,6 +82,9 @@
             " --input-file #{node['cluster']['cluster_config_path']}"
   end
 
+  # If defined in the config, retrieve a Custom Slurm Settings file from S3 and overrides the existing one
+  include_recipe 'aws-parallelcluster-slurm::retrieve_custom_settings_from_s3'
```

Comment:
I would generalize the name, even by supporting HTTPS alone we would support any file from the internet (provided it's reachable via HTTPS).

Reply:
done